source/_integrations/energenie_pimote.markdown

```diff
@@ -0,0 +1,27 @@
+---
+title: "Energenie PiMote"
+description: "Control your Energenie devices with a Raspberry Pi and Home Assistant"
+ha_release: "0.118"
+ha_category: Switch
+ha_iot_class: "Assumed State"
+ha_quality_scale: ~
+ha_config_flow: true
+ha_codeowners:
+  - '@matejmecka'
+ha_domain: energenie
+---
+### Energenie
+
+This integration allows users to use their Energenie Pimote devices to be controlled via Home Assistant.
+
+<div class='note'>
+This integration requires the Pimote addon installed on the Raspberry pi, along with previous setup!
```

Comment:
previous setup?

Reply:
What do you think is better way to reference it? 

`This integration requires having your switch previously assigned an ID from the link below?`

`This integration requires having followed the integration paired by giving it an ID which you can find below`

Reply:
I'm confused by a previous setup as a whole?

What if I don't have a previous setup? Does this mean there is a prerequisite?

Reply:
The prerequisite would be assigning an ID to the plug the person owns, that the Energenie addon would know and communicate with

Reply:
So, maybe I'm not understanding it or you are missing my point.

If a random new user gets here, never used anything of this before and reads: "along with previous setup" he will magically know what a previous setup is?

Reply:
I get it now. i'll fix it